api/api/spec/spec.yaml

```diff
@@ -4458,6 +4458,10 @@ components:
     SyscheckDatabase:
       type: object
       properties:
+        changes:
+          type: integer
+          format: int31
```

Comment:
```suggestion
          format: int32
```

Reply:
Typo fixed in 91e9e517832379241d788ee62026f06b26a36627.